spec/dummy/bin/setup

```diff
@@ -20,13 +20,16 @@ chdir APP_ROOT do
   # Install JavaScript dependencies if using Yarn
   # system('bin/yarn')
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?('config/database.yml')
-  #   cp 'config/database.yml.sample', 'config/database.yml'
-  # end
+  puts "\n== Copying sample files =="
+  unless File.exist?('config/database.yml')
+    cp 'config/database.sample.yml', 'config/database.yml'
+  end
 
   puts "\n== Preparing database =="
-  system! 'bin/rails db:setup'
+  system! 'bin/rails db:create'
+  system! 'bin/rails db:migrate'
+  system! 'bin/rails db:schema:load'
```

Comment:
Do we need to do this if we just migrated?

Reply:
yes it was still needed, db:migrate does not do that, I can also replace the schema:load/seed with db:setup as an option.